src/pages/[platform]/build-a-backend/data/working-with-files/index.mdx

```diff
@@ -252,6 +313,27 @@ There are three common deletion workflows when working with Storage files and th
 
 The following example removes the file association from the record, but does not delete the file from S3, nor the record from the database.
 
+<InlineFilter filters={["swift"]}>
+```swift title="ContentView"
+// Get the song record
+var song = try await Amplify.API.query(request: .get(Song.self, byId: currentSong.id)).get()
+
+// If the record has no associated file, we can return early.
+guard song?.coverArtPath != nil else {
+    print("There is no cover art key to remove image association")
+    return
+}
+
+// Remove associated file from record
+song?.coverArtPath = nil
+
+let updateResult = try await Amplify.API.mutate(request: .update(song!)).get()
```

Comment:
previously we have a guard on song
```
guard var song = queriedSong else {
    print("Song may have been deleted, no song by id: ", currentSong.id)
    return
}

```
now this will crash on line 330 since song is being force unwrapped if the query returned no song